Startup.cs

```diff
@@ -71,6 +72,13 @@ public void ConfigureServices(IServiceCollection services)
                             "DVT/1.0 (fredrik.malmo@icloud.com)");
                     }
                 );
+                services.AddHttpClient<ITaxAssessmentService, TaxAssessmentService>(client =>
+                {
+                    client.DefaultRequestHeaders.Accept.Add(
+                        new MediaTypeWithQualityHeaderValue("application/json"));
+                    client.DefaultRequestHeaders.UserAgent.ParseAdd(
+                        "DVT/1.0 (fredrik.malmo@icloud.com)");
```

Comment:
Does SSB require an email address in the user agent? If not, I'd appreciate it if you could leave my email address out of it (and remove it in PopulationInNorwayService).

Reply:
Consult the API documentation for requirements.

Reply:
I don't think they do, so I will remove it.